hazelcast/src/main/java/com/hazelcast/map/impl/mapstore/writebehind/WriteBehindStore.java

```diff
@@ -17,12 +17,12 @@
 package com.hazelcast.map.impl.mapstore.writebehind;
 
 import com.hazelcast.config.InMemoryFormat;
+import com.hazelcast.internal.serialization.SerializationService;
 import com.hazelcast.map.impl.MapStoreWrapper;
 import com.hazelcast.map.impl.mapstore.AbstractMapDataStore;
 import com.hazelcast.map.impl.mapstore.writebehind.entry.DelayedEntries;
 import com.hazelcast.map.impl.mapstore.writebehind.entry.DelayedEntry;
 import com.hazelcast.nio.serialization.Data;
-import com.hazelcast.internal.serialization.SerializationService;
 
```

Comment:
Another question: what happens when the puts arrive in a faster rate then you can get rid of them? Can it be that we get an OOME?